form.html

```diff
@@ -1,103 +1,58 @@
   <head>
-    <link rel="stylesheet" href="styles.css" />
-    <script src="index.js" defer></script>
+    <link rel="stylesheet" href="css/styles.css" />
+    <script src="form.js" defer></script>
+    <script src="menu.js" defer></script>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Quiz App</title>
   </head>
   <body>
     <header class="header">
-      <h1 aria-label="Quiz App">Quiz App</h1>
+      <h1 aria-label="Quiz App Form">Quiz App Form</h1>
     </header>
     <main class="container-padding">
-      <ul class="cards-list">
-        <li>
-          <article class="card">
-            <h2 class="content">What property flips the axes in flexbox?</h2>
-            <button class="card-answer">Show answer</button>
-            <p>flex-direction</p>
-            <ul class="tags">
-              <li><a class="hashtag-frontend" href="#test">#html</a></li>
-              <li><a class="hashtag-frontend" href="#test">#flexbox</a></li>
-              <li><a class="hashtag-frontend" href="#test">#css</a></li>
-            </ul>
-            <button
-              class="bookmark-button"
-              aria-label="Bookmark"
-              data-js="bookmark-button"
-            >
-              <img src="assets/bookmark.svg" alt="Bookmark" class="bookmark" />
-            </button>
-            <button
-              class="bookmark-button--filled"
-              aria-label="Bookmark filled"
-              data-js="bookmark-button--filled"
-            >
-              <img
-                src="assets/bookmark-filled.svg"
-                alt="Bookmark-filled"
-                class="bookmark--filled"
-              />
-            </button>
-          </article>
-        </li>
-        <li>
-          <article class="card">
-            <h2 class="content">Lorem Ipsum dolor sit?</h2>
-            <button class="card-answer">Show answer</button>
-            <p>flex-direction</p>
-            <ul class="tags">
-              <li><a class="hashtag-frontend" href="#test">#html</a></li>
-              <li><a class="hashtag-frontend" href="#test">#flexbox</a></li>
-              <li><a class="hashtag-frontend" href="#test">#css</a></li>
-            </ul>
-            <button
-              class="bookmark-button"
-              aria-label="Bookmark"
-              data-js="bookmark-button"
-            >
-              <img src="assets/bookmark.svg" alt="Bookmark" class="bookmark" />
-            </button>
-            <button
-              class="bookmark-button--filled"
-              aria-label="Bookmark filled"
-              data-js="bookmark-button--filled"
-            >
-              <img
-                src="assets/bookmark-filled.svg"
-                alt="Bookmark-filled"
-                class="bookmark--filled"
-              />
-            </button>
-          </article>
-        </li>
-      </ul>
+      <form class="question-form" id="question-form">
```

Comment:
please use data-js instead of id here instead

Reply:
Oh my god. For whatever reason, i didn´t think about it. I´m annoyed with myself and fix this. Thanks for your Feedback!